### PR TITLE
Added a script for running all unit tests; moved utility and test files out of bin

### PR DESCRIPTION
**Pull request by mitza-oci**

### CODE REVIEW

File: tools/scripts/unit_tests.pl [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env perl
```
Comment (jrw972) [inline tools/scripts/unit_tests.pl:1]: This should be in `DDS_ROOT/bin` or the other test related files should be moved to `DDS_ROOT/tools/scripts/unit_tests.pl`.

**Review by jrw972 — CHANGES_REQUESTED** [review]

File: tools/scripts/unit_tests.pl (hunk 1 above)
Comment (mitza-oci) [reply tools/scripts/unit_tests.pl:1]: I agree with the latter.  All this random stuff doesn't need to be added to the user's PATH.

**Comment by jwillemsen** [comment]
Moving the lst files also needs a change in ACE/TAO scripts, see https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/bin/diff-builds-and-group-fixed-tests-only.sh

**Comment by mitza-oci** [comment]
> 
> 
> Moving the lst files also needs a change in ACE/TAO scripts, see https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/bin/diff-builds-and-group-fixed-tests-only.sh

Seems like this would be a good opportunity to remove the reverse dependency.  Should the script be in the autobuild repository?

**Comment by jwillemsen** [comment]
> > Moving the lst files also needs a change in ACE/TAO scripts, see https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/bin/diff-builds-and-group-fixed-tests-only.sh
> 
> Seems like this would be a good opportunity to remove the reverse dependency. Should the script be in the autobuild repository?

Yes, that could be an alternative location.

**Review by jrw972 — APPROVED** [review]

**Comment by mitza-oci** [comment]
> 
> 
> > Moving the lst files also needs a change in ACE/TAO scripts, see https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/bin/diff-builds-and-group-fixed-tests-only.sh
> 
> Seems like this would be a good opportunity to remove the reverse dependency. Should the script be in the autobuild repository?

https://github.com/DOCGroup/ACE_TAO/pull/1429

**Pull request merged**